docs/guidance/publishing-from-portal.md

```diff
@@ -1,9 +1,98 @@
-# Using OCDS + Project Level Data Spec to publish from an infrastructure transparency portal
+# Publishing data from an infrastructure transparency portal
+
+The project level data specification can be used to publish standardised open data on infrastructure projects where information is already collected and disclosed through infrastructure transparency portals, whether CoST Multi-Stakeholder Groups, government agencies or civil society organizations.
+
+Publishing standardised open data reduces barriers to use of data and supports the development of reusable tools and methodologies for working with data on infrastructure projects.
+
+If you also collect detailed data on contracting processes, this can be published using the [Open Contracting Data Standard (OCDS)](http://standard.open-contracting.org/latest/en/).
+
+```eval_rst
+.. admonition:: Linking to related information
+    :class: Tip
+
+    .. markdown::
+
+      Infrastructure transparency portal creators should consider what other types of information might be important to citizens, in addition to the in depth scrutiny related information in the project level data specification.
+
+      For example, [Highways England](https://highwaysengland.co.uk/roads/) provides links to congestion and traffic restriction information alongside information on roads projects.
+
+```
+
+## Getting started
+
+*Some of the following steps may require support from a technical expert. You can also contact the OCDS Helpdesk (<a href="mailto:data@open-contracting.org">data@open-contracting.org</a>) for guidance.*
+
+### (1) Make a commitment
+
+Consider making or advocating for a public commitment to publish standardised open data using the project level data specification and OCDS.
+
+Commitments are important to help align implementation with the goals of publishing open data and to help overcome technical, political or bureaucratic barriers to publication.
+
+Applications to join [CoST](http://infrastructuretransparency.org/) can be used to make a commitment or if your country is a member of the [Open Government Partnership](https://www.opengovpartnership.org/), your National Action Plan is another great place to start.
+
+Refer to the [OCDS implementation journey](https://www.open-contracting.org/implement/#/1) for information and resources about making commitments related to OCDS.
+
+### (2a) Map project level data and summary contracting process data
+
+Map existing data structures to the [project level data specification](../projects/index.md).
+
+```eval_rst
+.. admonition:: Tip
+    :class: Tip
+
+    .. markdown::
+
+      The [field level mapping template](https://docs.google.com/spreadsheets/d/1xHLf_w193pp97zfzhLc_LI-yEXrR_eyscga06Qo1blk/copy) can be used to document your mapping.
```

Comment:
Hyphenate "field-level".

Reply:
To avoid confusion with the core mapping template, please refer to this as the "OC and Infrastructure Field-Level Mapping Template"